### DISCUSSION TITLE
The presentation parsing method does not support "Enhancing RAPTOR strategy with Recalls"

### DISCUSSION THREAD

**Discussion by peizimo** [infiniflow/ragflow#2827] [Q&A]
The presentation parsing method does not support "Enhancing RAPTOR strategy with Recalls"，Ask for help
![84cdc8f8d56c40104830f8dc8d32e6f7](https://github.com/user-attachments/assets/f69ef24e-514f-4b6b-bf1e-0cd701b4ba65)

**Comment by KevinHuSh** [comment]
Fixed!
